_search-plugins/concurrent-segment-search.md

```diff
@@ -156,7 +150,30 @@ Additionally, some [Profile API]({{site.url}}{{site.baseurl}}/api-reference/prof
 
 ## Limitations
 
-Parent aggregations on [join]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) fields do not support the concurrent search model. Thus, if a search request contains a parent aggregation, the aggregation will be executed using the non-concurrent path even if concurrent segment search is enabled at the cluster level.
+The following aggregations do not support the concurrent search model. If a search request contains one of these aggregations, the request will be executed using the non-concurrent path even if concurrent segment search is enabled at the cluster level or index level.
+- Parent aggregations on [join]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) fields. See [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/9316).
+- Sampler and Diversified Sampler aggregations. See [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/110750).
```

Comment:
Confirm that "Diversified Sampler" is meant to be capitalized and should not be formatted differently.

Reply:
Or is it a common noun and should just be "diversified sampler"?

Reply:
The name of the aggregation type is "Diversified sampler". Similar to "Significant terms" vs. "Terms" aggregations.

See: https://opensearch.org/docs/latest/aggregations/bucket/diversified-sampler/

Reply:
Got it, it should be in code font, then.